src/nxdoc/tutorials/educational/platform-as-a-service.md

```diff
@@ -262,25 +263,16 @@ Because lightweight containers are cheaper to create or to shutdown, this 2-leve
 To manage this container factory `nuxeo.io` is based on several key components:
 
 *   Docker: as the lightweight container system
-*   CoreOS: as system host for running the Docker container
-    *   Fleet: as manager to distribute services across the cluster
-    *   etcd: as distributed registry
-*   Gogeta: as dynamic load balancer
-*   nuxeo.io manager: the Nuxeo-based application used to control the whole system
+*   Rancher: as the containers orchestation layer.
+*   etcd: as distributed registry
+*   XXX: as dynamic load balancer
+*   arken: the Nuxeo-based layer to control deployments of Nuxeo Applications in there
+*   nuxeo.io manager: a web ui to offer ability to start / stop instances from a user-friendly web interface.
 
 ![](https://www.lucidchart.com/publicSegments/view/5450599b-5a40-4a34-8243-49f40a005489/image.png)
```

Comment:
@aescaffre we have infrastructure level schema, but I wonder if it's worth putting detail about that in the documentation. Nuxeo.io doesn't exist anymore, Nuxeo Cloud doesn't exist yet. I'm not sure we'll want to put too much details on the Nuxeo Cloud infrastructure and internals for obvious security reasons.